pkg/secrethub/credentials/readers.go

```diff
@@ -1,52 +1,98 @@
 package credentials
 
 import (
+	"fmt"
 	"io/ioutil"
 	"os"
 )
 
-// Reader helps with reading bytes from a configured source.
-type Reader interface {
```

Comment:
Should we keep `Reader` around for a bit for backwards compatibility?

Reply:
Yes, good idea.